### DISCUSSION TITLE
Handle link clicks

### DISCUSSION THREAD

**Discussion by lublak** [ONLYOFFICE/DocumentServer#1681] [Q&A] [answered]
Currently we need to catch all link clicks.
https://api.onlyoffice.com/editors/config/events
Unfortunately I can't find an event for onLinkClick.
Is there another way to catch this?

**Comment by ShockwaveNN** [comment, marked as answer]
I don't think there is a way to do so

Could you describe in more details why you need that so I can send this info to our dev team?

**Reply by lublak** [reply]
@ShockwaveNN

The links themselves are not accessible from the current server.
But they are relative.
`../somefile.html`
If a person clicks on a link, firstly no dialog should appear, because in the internet all links can be considered safe and secondly I have to open another page:
http://someotherserver?open=../somefile.html
not the same server:
http://onlyoffice/7.0.1-37/web-apps/apps/spreadsheeteditor/somefile.html

**Reply by ShockwaveNN** [reply]
I've shown this info to our dev team and they'll decide if we need that

If you really need this feature - better to contact sales@onlyoffice.com or support@onlyoffice.com

**Reply by lublak** [reply]
@ShockwaveNN I found out that I could rewrite the links with a plugin: https://api.onlyoffice.com/plugin/basic
Only the dialog always opens (do you really want to open this link?) but I'm ok with that.

**Reply by ShockwaveNN** [reply]
@lublak Yeah, dev team told me that plugin be suited for you, but not sure it will work flawlessly until you try it

**Reply by lublak** [reply]
@ShockwaveNN it works for me. 
But there are usually only 1-3 links per document. Of course, with more links there could be more problems.
But since we don't have more than 1-3 links per document, I don't have to worry about that.
Thanks in any case for the answers :)

**Reply by ShockwaveNN** [reply]
@lublak Glad I've helped you
